packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -16,7 +16,16 @@ let babelHelpers = eval(buildExternalHelpers(null, "var"));
 function wrapPackagesArray(type, names) {
   return (names || []).map(function (val) {
     if (typeof val === "string") val = [val];
-    val[0] = __dirname + "/../../babel-" + type + "-" + val[0];
+
+    // relative path (outside of monorepo)
+    if (val[0][0] === ".") {
+      val[0] = __dirname + "/" + val[0];
```

Comment:
If this is a relative path, shouldn't we be making it relative to the `options.json` file? This makes it relative to this `index.js` which doesn't seem right.